File: RentAPI/Models/Cliente.cs
namespace RentAPI.Models
{
    public class Cliente
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Telefone { get; set; }
        public string Endereco { get; set; }
        public string Bairro { get; set; }
        public string Cidade { get; set; }
        public string Estado { get; set; }
        public string Email { get; set; }
        public string Documento { get; set; }
        public decimal ValorContrato { get; set; }
        public DateOnly InicioContrato { get; set; }
        public DateOnly TerminoContrato { get; set; }
        public int DiaVencimento { get; set; }
        public string UltimoPgto { get; set; }

        public Cliente() { }

    }

}

File: rentApp/src/models/cliente.ts
export class Cliente {
  id: number;
  nome: string;
  telefone: string;
  endereco: string;
  bairro: string;
  cidade: string;
  estado: string;
  email: string;
  documento: string;
  valorContrato: number;
  inicioContrato: Date;
  terminoContrato: Date;
  diaVencimento: number;
  ultimoPgto: string;

  constructor(
    id: number = 0,
    nome: string = '',
    telefone: string = '',
    endereco: string = '',
    bairro: string = '',
    cidade: string = '',
    estado: string = '',
    email: string = '',
    documento: string = '',
    valorContrato: number = 0,
    inicioContrato: Date = new Date(),
    terminoContrato: Date = new Date(),
    diaVencimento: number = 0,
    ultimoPgto: string = ''
  ) {
    this.id = 0;
    this.nome = nome;
    this.telefone = telefone;
    this.endereco = endereco;
    this.bairro = bairro;
    this.cidade = cidade;
    this.estado = estado;
    this.email = email;
    this.documento = documento;
    this.valorContrato = valorContrato;
    this.inicioContrato = inicioContrato;
    this.terminoContrato = terminoContrato;
    this.diaVencimento = diaVencimento;
    this.ultimoPgto = ultimoPgto;
  }
}
